librz/include/rz_bin.h

```diff
@@ -819,6 +819,11 @@ typedef struct rz_bin_resource_t {
 	char *language;
 } RzBinResource;
 
+typedef struct rz_bin_golang_info_t {
+	char *version; ///< golang version
+	char *settings; ///< golang build settings
+} RzBinGoInfo;
+
```

Comment:
Yes I agree with @XVilka . I think we should definitely find another way to handle this in a more generic way. I don't like making this part of an API. Is it used only within RzBin? If so, we can move it in a separate header e.g. `bin_private.h`.

Reply:
i think this is a useful api, since it is used by multiple plugins. therefore it should stay in rz_bin.h